osquery/tables/events/linux/process_events.cpp

```diff
@@ -81,7 +81,9 @@ bool ProcessUpdate(size_t type, const AuditFields& fields, AuditFields& r) {
   }
 
   if (type == AUDIT_PATH) {
-    r["mode"] = (fields.count("mode")) ? fields.at("mode") : "";
+    std::stringstream ss;
+    ss << "0" << std::oct << fields.at("mode");
```

Comment:
Be careful, in the previous code the existence of `mode` was checked before using it. If `mode` does not exist as a key in the `map` then this new implementation will throw an exception.